Classes/Controller/Cart/OrderController.php

```diff
@@ -83,7 +85,37 @@ public function createAction(
             return $this->redirect('show', 'Cart\Cart');
         }
 
-        $this->eventDispatcher->dispatch(new ProcessOrderCheckStockEvent($this->cart));
+        $processOrderCheckStockEvent = new ProcessOrderCheckStockEvent($this->cart);
+        $this->eventDispatcher->dispatch($processOrderCheckStockEvent);
+
+        if (!$processOrderCheckStockEvent->isEveryProductAvailable()) {
+            $errors = $processOrderCheckStockEvent->getInsufficientStockMessages();
```

Comment:
```suggestion
            $insufficientStockMessages = $processOrderCheckStockEvent->getInsufficientStockMessages();
```

Perhaps a variable name is not necessary because it's used one time only.

Reply:
Done.